core/templates/dev/head/components/forms/AudioTranslationsEditorDirective.js

```diff
@@ -126,8 +127,9 @@ oppia.directive('audioTranslationsEditor', [
                   $scope.errorMessage = null;
                   $scope.saveButtonText = BUTTON_TEXT_SAVE;
                   $scope.saveInProgress = false;
-
-                  $scope.languageCode = allowedAudioLanguageCodes[0];
+                  $scope.languageCode =
+                  allowedAudioLanguageCodes.indexOf(prevLanguageCode) != -1 ?
```

Comment:
Indent this line and the following line by 2 spaces.

Reply:
Done.